LayoutTests/transitions/flex-transitions.html

```diff
@@ -20,31 +21,31 @@
 }
 
 #flex-row div {
-    flex: 1 0 0px;
+    -webkit-flex: 100 0 0px;
 }
 .final #row1 {
-    flex: 3 0 0px;
+    -webkit-flex: 300 0 0px;
 }
 
 #flex-column {
-    flex-direction: column;
+    -webkit-flex-direction: column;
 }
 #flex-column div {
-    flex: 1 0 0px;
+    -webkit-flex: 100 0 0px;
 }
 .final #column1 {
-    flex: 0 0 20px;
+    -webkit-flex: 0 0 200px;
 }
 
 #flex-negative div {
-    flex: 1 1 50px;
+    -webkit-flex: 100 100 50px;
 }
 .final #negative1 {
-    flex: 1 100px;
+    -webkit-flex: 100 100 100px;
 }
```

Comment:
Is there a reason to use the prefixed version here?

Reply:
I personally had no specific reason but just copied Blink's commit. I think I can rename this test to have 'prefixes' and then another test without? It would give coverage to both. Any input? @nt1m

Reply:
The prefixed versions are just aliases of the unprefixed ones, I'd just remove them. There's not much point testing aliases in general.